src/bebone/assets/image/image.h

```diff
@@ -6,8 +6,7 @@
 
 #include <iostream>
 
-#include "../../core/types.h"
-#include "../../core/noncopyable.h"
```

Comment:
If you removed the noncopyable.h include here, why didn't you added it to core.h?